docs/model-deployment-language-wrapper-python.md

```diff
@@ -74,61 +76,59 @@ s2i usage seldonio/seldon-core-s2i-python3:0.18
             return X
     ```
 
-    - File and class name `MyModel` should be the same as **MODEL_NAME** under `.s2i/environment`
+    - File and class name `MyModel` should be the same as **MODEL_NAME** in `Dockerfile`
     - Load or initiate your model under the `__init__` function
     - The predict method takes a numpy-array `X` and list of string `feature_names` (optional), then returns an array of predictions (the return array should be at least 2-dimensional)
 
     More detailed information on how to write the Python file for model deployment in different frameworks, please refer to the section [Example Codes for Different Frameworks](#optional-example-codes-for-different-frameworks).
 
 ## Build the Image
 
-Make sure you are in the folder that includes `requirements.txt`, `.s2i/environment`, `python file for model deployment`, and `model file`...etc.
-
-If this folder is managed by `Git`, please commit all changes into the git.
+- Make sure you are in the folder that includes `requirements.txt`, `Dockerfile`, `python file for model deployment`, and `model file`.
 
-We will use `seldonio/seldon-core-s2i-python3:0.18` as a base image, installing environment and packaging our model file into the target, `my-model-image`. You can use the following command to package the docker image:
+- If this folder is managed by `Git`, please commit all changes into the git.
```

Comment:
I think if we are using docker, this is not required.

Reply:
ok, updated